bedrock/mozorg/templates/mozorg/contribute/studentambassadors/join.html

```diff
@@ -29,7 +29,7 @@
 
 {% block billboardcontent %}
   <form method="POST" class="billboard {% if form.non_field_errors() or form.errors %}has-errors{% endif %}"
-        id="ambassadors-form" action="{{ secure_url() }}#ambassadors-form">
+        id="ambassadors-form" action="{{ request.build_absolute_uri(request.path) }}#ambassadors-form">
```

Comment:
Not that I can see. Original intent was to mimic the functionality previously provided by `secure_url`, but agree that it would be more straightforward to replace with the `url` helper. Will update where possible.